config/initializers/mailer.rb

```diff
@@ -13,3 +13,12 @@
   host: DEFAULT_URL_HOST,
   port: ENV['DEFAULT_URL_PORT'] || default_port
 }
+
+unless ENV['DISABLE_SANDBOX_WARNING'] == 'true'
+  class PrefixEmailSubject
+    def self.delivering_email(mail)
+      mail.subject = "[TEST] " + mail.subject
+    end
+  end
+  ActionMailer::Base.register_interceptor(PrefixEmailSubject)
+end
```

Comment:
Faaaancy. Maybe worth adding a little comment above saying something like "indicate that this is not a real request in the email subjects".